frontend/test/pytest/test_custom_devices.py

```diff
@@ -22,6 +22,7 @@
 from catalyst.compiler import get_lib_path
 from catalyst.device import extract_backend_info, get_device_capabilities
 from catalyst.utils.exceptions import CompileError
+from frontend.test.conftest import CONFIG_CUSTOM_DEVICE
```

Comment:
This import looks a bit strange to me, does that work generally? It seems like it might only work if launching python from the root directory

Reply:
Let's go with a safer solution. Thanks for pointing this out.